code/src/containers/tables/orders-table/components/orders-table-head/OrdersTableHead.tsx

```diff
@@ -30,29 +30,45 @@ const OrdersTableHead = ({ head }: OrderTableHeadProps) => {
 
   const sortKey = getSortKey(head);
 
+  const isActive = searchParams.get("sort")?.startsWith(sortKey);
+
+  const sortDirection = isActive
+    ? sortDirections[sortKey]
+    : initialSortOrder[sortKey];
+
   const handleSort = () => {
-    if (sortKey) {
-      const newSortDirection =
-        sortDirections[sortKey] === "desc" ? "asc" : "desc";
-
-      setSortDirections((prevDirections) => ({
-        ...prevDirections,
-        [sortKey]: newSortDirection
-      }));
-
-      const newSort = `${sortKey},${newSortDirection}`;
-      const params = new URLSearchParams(searchParams);
-      params.set("sort", newSort);
-      setSearchParams(params);
+    let newSortDirection: SortOrder;
+
+    if (isActive) {
+      newSortDirection = sortDirections[sortKey] === "desc" ? "asc" : "desc";
+    } else {
+      newSortDirection = initialSortOrder[sortKey] === "desc" ? "asc" : "desc";
```

Comment:
You can refactor it to smth like that
`newSortDirection = (isActive ? sortDirections[sortKey] : initialSortOrder[sortKey]) === "desc" ? "asc" : "desc";`

Reply:
Done! Thanks